.expeditor/scripts/shared.ps1

```diff
@@ -78,9 +78,20 @@ function Install-Rustup($Toolchain) {
   } else {
       Write-Host "Installing rustup and $toolchain-x86_64-pc-windows-msvc Rust."
       [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
-      invoke-restmethod -usebasicparsing 'https://static.rust-lang.org/rustup/dist/i686-pc-windows-gnu/rustup-init.exe' -outfile 'rustup-init.exe'
-      ./rustup-init.exe -y --default-toolchain $toolchain-x86_64-pc-windows-msvc --no-modify-path --profile=minimal
-      $env:path += ";$env:USERPROFILE\.cargo\bin"
+      try { 
+        Invoke-RestMethod -UseBasicParsing 'https://static.rust-lang.org/rustup/dist/i686-pc-windows-gnu/rustup-init.exe' `
+                          -OutFile 'rustup-init.exe' `
+                          -MaximumRetryCount 5 `
+                          -RetryIntervalSec 5 
```

Comment:
Might be worth a comment about why this particular call is getting the special treatment.